static/css/error.less

```diff
@@ -0,0 +1,61 @@
+/* ==========================================================================
+   cfgov-refresh
+   Page-specific styles for the error pages.
+   ========================================================================== */
+
+.btn__error-home {
+    .respond-to-max(@tablet-min, {
+        width: 100%;
+        text-align: center;
+    });
+
+    .respond-to-max(600px, {
+        .grid_column(12);
+    });
+}
+
+.error_action_container {
+    background-color: @white;    
+}
+
+.error_copy_container {
+    .grid_column(6);
+
+    .respond-to-range(601px, 1199px, {
+        .grid_column(8);
+    });
+
+    .respond-to-max(600px, {
+        .grid_column(12);
+    });
+}
+
+.error_featured-img {
```

Comment:
Is it possible to set the bg-image on the content wrapper rather than deal with absolute positioning images?